src/backend/base/langflow/components/tools/file_manipulation.py

```
@@ -0,0 +1,1322 @@
+                line = lines[i].rstrip("\n")
+                result += f"{i + 1:4}: {line}\n"
+
+            # Add truncation notice if needed (only for default preview mode)
+            if line_number is None and len(lines) > preview_length:
+                result += f"\n... and {len(lines) - preview_length} more lines"
+        except OSError as e:
+            return f"Error getting file preview: {e!s}"
+        else:
+            return result
+
+    def _edit_file(
```

Pylint (3.3.7)

[refactor] Too many arguments (14/5)

(R0913)